lib/inventoryware/utils.rb

```diff
@@ -59,12 +59,7 @@ def self.check_file_location?(path)
 
     # raise an error if given path isn't a directory
     def self.exit_unless_dir(path)
```

Comment:
This name isn't true anymore </pedantry>.

Reply:
This whole section of code is a bit over kill. It's all be removed/replaced with a single `FileUtils.mkdir_p` via the `Config`

Reply:
The new commit looks much better.  Thanks.